src/modules/Todo.js

```diff
@@ -0,0 +1,67 @@
+export default class Todo {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) ?? [];
+    this.tasks.sort((a, b) => a.index - b.index);
+  }
+
+  static toUpper(string) {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
+  get() {
+    return this.tasks;
+  }
+
+  set(tasks) {
+    this.tasks = tasks;
+    return this;
+  }
+
+  add(description) {
+    (this.tasks).push({
+      description: Todo.toUpper(description),
+      completed: false,
+      index: (this.tasks.length) + 1,
+    });
+    this.save();
+    return this;
+  }
```

Comment:
- [x] -It would be better If you add a function to check for empty inputs.

<img width="382" alt="Screenshot 2023-01-24 at 13 23 27" src="https://user-images.githubusercontent.com/113255559/214293479-3d3e24c4-8681-4cb1-a529-778cb4273c57.png">

Reply:
Thank You @Ouail-01 , I will make the changes and request a new review